src/sema/ast.rs

```diff
@@ -768,7 +768,7 @@ pub struct Contract {
     pub fixed_layout_size: BigInt,
     pub functions: Vec<usize>,
     pub all_functions: BTreeMap<usize, usize>,
-    pub virtual_functions: HashMap<String, usize>,
+    pub virtual_functions: HashMap<String, Vec<usize>>,
```

Comment:
This needs a very clear comment explaining what this `HashMap<..>` does and what the `Vec<usize>` means